File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/SignDB/Sign/SignForm.cs
using Serenity.ComponentModel;
using System;

namespace CorrespondenceSystem.SignDB.Forms;

[FormScript("SignDB.Sign")]
[BasedOnRow(typeof(SignRow), CheckNames = true)]
public class SignForm
{
    public string Title { get; set; }
    public string SignAttachment { get; set; }
}

File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/ServerTypes/SignDB/SignForm.ts
import { StringEditor, ImageUploadEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface SignForm {
    Title: StringEditor;
    SignAttachment: ImageUploadEditor;
}

export class SignForm extends PrefixedContext {
    static readonly formKey = 'SignDB.Sign';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!SignForm.init)  {
            SignForm.init = true;

            var w0 = StringEditor;
            var w1 = ImageUploadEditor;

            initFormType(SignForm, [
                'Title', w0,
                'SignAttachment', w1
            ]);
        }
    }
}